platform/src/Panel.js

```diff
@@ -86,6 +92,8 @@ class Panel {
 
     setValue(value) {
         this.editor.setValue((value+""), 1);
+        // Reset undo manager
+        this.editor.session.getUndoManager().markClean();
```

Comment:
This could be made to reset() rather than markclean() so that undo doesn't allow users to undo back to a blank panel as setValue is only used by the platform to populate the contents. Alternatively, we could add a method for resetting the undos to allow the platform to manage this is we want finer-grained control of this.